client/src/pages/SymptomsCheck.js

```diff
@@ -305,16 +305,9 @@ const Step7 = (props) => {
 export const SymptomsCheck = () => {
   const [state, setState] = useState(INITIAL_STATE);
   const updateAnswers = (key, value) => {
-    const { answers } = state;
-    const updatedAnswers = { ...answers, [key]: value };
-    setState({ ...state, updatedAnswers });
-    if (key === "careFacility") {
-      localStorage.setItem(
-        "symptomsCheckAnswers",
-        JSON.stringify(updatedAnswers),
-      );
-    }
+    setState({ ...state, [key]: value });
   };
+  localStorage.setItem("symptomsCheckAnswers", JSON.stringify(state));
```

Comment:
Hmm looks good that you simplified the `updateAnswers` logic because we only currently need the answers "map" ( could also use Map() instead of object) but updating local storage here on every render of the page seems wrong?
Before I only did this at the last step which was also kind of a temporary solution, but this might not be good either, maybe update after every call to `updateAnswers`? 
And since it's a side effect it probably belongs in a `useEffect()` hook https://egghead.io/lessons/react-store-values-in-localstorage-with-the-react-useeffect-hook

Reply:
@robinv85 , do we even need the localstorage bit? Was thinking that since data is being stored in state, we could do away with this. Your thoughts?